test/ILLink.RoslynAnalyzer.Tests/RequiresUnreferencedCodeAnalyzerTests.cs

```diff
@@ -36,6 +71,85 @@ class C
 				VerifyCS.Diagnostic ().WithSpan (8, 17, 8, 21).WithArguments ("C.M1()", "message", ""));
 		}
 
+		[Fact]
+		public async Task SimpleDiagnosticFix ()
+		{
+			var test = @"
+using System.Diagnostics.CodeAnalysis;
+
+public class C
+{
+    [RequiresUnreferencedCodeAttribute(""message"")]
+    public int M1() => 0;
+
+    int M2() => M1();
+}
+class D
+{
+    public int M3(C c) => c.M1();
+
+    public class E
+    {
+        public int M4(C c) => c.M1();
+    }
+}
+public class E
+{
+    public class F
+    {
+        public int M5(C c) => c.M1();
+    }
+}
+";
+
+			var fixtest = @"
+using System.Diagnostics.CodeAnalysis;
+
+public class C
+{
+    [RequiresUnreferencedCodeAttribute(""message"")]
+    public int M1() => 0;
+
+    [RequiresUnreferencedCode(""calls M1"")]
+    int M2() => M1();
+}
+class D
+{
+    [RequiresUnreferencedCode(""calls M1"")]
+    public int M3(C c) => c.M1();
+
+    public class E
+    {
+        [RequiresUnreferencedCode(""calls M1"")]
+        public int M4(C c) => c.M1();
+    }
+}
+public class E
+{
+    public class F
+    {
+        [RequiresUnreferencedCode()]
+        public int M5(C c) => c.M1();
+    }
+}
+";
+
+			await VerifyRequiresUnreferencedCodeCodeFix (test, fixtest, new[] {
+	// /0/Test0.cs(9,17): warning IL2026: Using method 'C.M1()' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code. message.
```

Comment:
Nit: We should delete the name of the file /0/Tests0.cs to be consistent with the other tests

Reply:
This seems to be the format written by the verifier utils -- I'd rather find some way to change that at the root. Otherwise we'll just be constantly fighting the tooling.